seeds/01_consent.ts

```
@@ -0,0 +1,72 @@
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Ahan Gupta <ahangupta.96@gmail.com>
+
+ --------------
+ ******/
+
+'use strict'
+import * as Knex from 'knex'
+
+const consents = [
+  {
+    id: '123',
+    initiatorId: 'PISPA',
+    participantId: 'DFSPA',
+    credentialId: null,
+    credentialType: null,
+    credentialStatus: null,
+    credentialPayload: null,
+    credentialChallenge: null
+  },
+  {
+    id: '124',
+    initiatorId: 'PISPB',
+    participantId: 'DFSPA',
+    credentialId: '9876',
+    credentialType: 'FIDO',
+    credentialStatus: 'PENDING',
+    credentialPayload: null,
+    credentialChallenge: 'string_representing_challenge_a'
+  },
+  {
+    id: '125',
+    initiatorId: 'PISPC',
+    participantId: 'DFSPA',
+    credentialId: '9875',
+    credentialType: 'FIDO',
+    credentialStatus: 'VERIFIED',
+    credentialPayload: 'string_representing_public_key_a',
+    credentialChallenge: 'string_representing_challenge_b'
+  }
+]
+
+exports.seed = function (knex: Knex): Knex.QueryBuilder<number[]> | number {
+  try {
+    return knex('Consent').insert(consents)
+  } catch (err) {
+    if (err.code === 'ER_DUP_ENTRY') return -1001
+    else {
+      console.log(`Uploading seeds for consents has failed with the following error: ${err}`)
+      return -1000
```

This should probably throw a `new Error()` error instead of returning a number.